.kiro/steering/nativeapp.structure.md

```
@@ -0,0 +1,35 @@
+# Native App Structure (apps/nativeapp)
+
+## Directory Layout
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)